README.md

```diff
@@ -45,4 +45,77 @@ http://localhost:9000/login
 1. Run `sbt clean release`
 2. Answer the questions
 3. Login to https://oss.sonatype.org/ then Close, and Release the Staging Repository
-   25 
\ No newline at end of file
+   25 
+
+## Creating a deployment and running it as a Linux service
+
+1. To create a deployable zip package of the application run:
+```
+sbt dist
+```
+2. Copy the zip file create in #1 to the deployment directory, e.g.
+```
+sudo unzip ./target/universal/ctd-omega-editorial-frontend-0.1.0-SNAPSHOT.zip -d /opt
+```
+3. Create a ctd-web user which is specifically used to run the service:
+```
+sudo adduser --system --shell /sbin/nologin ctd-web
+```
+4. Change the owner of the deployment directory to be the newly created ctd-web user:
+```
+sudo chown -R ctd-web:ctd-web /opt/ctd-omega-editorial-frontend-0.1.0-SNAPSHOT
+```
+5. It is recommended to create a symbolc link to the current version as follows:
+```
+sudo ln -s /opt/ctd-omega-editorial-frontend-0.1.0-SNAPSHOT /opt/ctd-omega-editorial-frontend
+```
+6. Create a systemd config file as follows, replacing the text `some_application_secret` with a specially generated secret, as described at https://www.playframework.com/documentation/2.8.x/ApplicationSecret#Generating-an-application-secret
+```
+[Unit]
+Description=CTD Omega Editorial Frontend
+Documentation=https://github.com/nationalarchives/ctd-omega-editorial-frontend
+After=syslog.target
+
+[Service]
+Type=simple
+User=ctd-web
+Group=ctd-web
+Environment="APPLICATION_SECRET=some_application_secret"
+ExecStart=/opt/ctd-omega-editorial-frontend/bin/ctd-omega-editorial-frontend
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+```
+7. Save the config file as ctd-omega-editorial-frontend.service in the `/etc/systemd/system` directory and ensure that the file is executable by the ctd-user, e.g.
```

Comment:
I'd surround ctd-omega-editorial-frontend.service with back ticks, as you've done with `/etc/systemd/system`.

Reply:
Confirmed.